tests/bundler.js

```diff
@@ -19,15 +20,17 @@ const metadata = {
 	license: 'MIT',
 	version: '9.0.1'
 };
-const bundlerConfig = Object.assign( {}, metadata, {
-	src: 'tests/fixtures/testPackage/src/index.js',
-	dist: {
-		esm: 'tests/fixtures/testPackage/dist/es2015.js',
-		cjs: 'tests/fixtures/testPackage/dist/es5.js'
-	}
-} );
+const bundlerConfig = ( packageName = 'testPackage' ) => {
+	return Object.assign( {}, metadata, {
+		src: `tests/fixtures/${packageName}/src/index.js`,
+		dist: {
+			esm: `tests/fixtures/${packageName}/dist/es2015.js`,
+			cjs: `tests/fixtures/${packageName}/dist/es5.js`
+		}
+	} );
+};
 
-describe( 'bundler', () => {
+xdescribe( 'bundler/testPackage', () => {
```

Comment:
It shouldn't be changed to `xdescribe`. Additionally I have an unwritten convention (that I should probably write down :D) that every module has one big `describe` with the name of the module. I'd probably nest some `describe`s:

```javascript
describe( 'bundler', () => {
	describe( 'testPackage', () => {
		// …
	} );

	describe( 'jsonPackage', () => {
		// …
	} );
} );
```

Reply:
Actually it seems that tests can use some refactoring, so probably it will be changed later anyway. The only blocking issue is the test for the feature itself.